frontend/src/services/websocket/client.ts

```
@@ -0,0 +1,141 @@
+      ...config,
+    };
+  }
+
+  connect(): void {
+    if (this.socket?.connected) {
+      return;
+    }
+
+    const token = localStorage.getItem('authToken');
+
+    this.socket = io(this.config.url!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)